apm-sniffer/apm-sdk-plugin/dubbo-2.7.x-plugin/src/main/java/org/apache/skywalking/apm/plugin/asf/dubbo/DubboInterceptor.java

```diff
@@ -136,6 +137,9 @@ private void dealException(Throwable throwable) {
      */
     private String generateOperationName(URL requestURL, Invocation invocation) {
         StringBuilder operationName = new StringBuilder();
+        String groupStr = requestURL.getParameter(Constants.GROUP_KEY);
+        groupStr = null == groupStr ? "" : groupStr + " ";
+        operationName.append(groupStr);
```

Comment:
I am a little curious about, why choose ` `(whitespace) as the separator?

Reply:
Which separator is better?

Reply:
I can't answer this directly because you are asking for this. We usually prefer `.` and `/` as separator. Do you prefer `groupname`.`classname`.`methodName`.

Reply:
/ is ok

Reply:
I prefer groupname/classname.methodName

Reply:
OK, go for it.